criu/crtools.c

```diff
@@ -541,6 +541,12 @@ int main(int argc, char *argv[], char *envp[])
 	       "                        pages images of previous dump\n"
 	       "                        when used on restore, as soon as page is restored, it\n"
 	       "                        will be punched from the image\n"
+	       "  --zero-pages          don't dump pages containing only zero bytes. This is a\n"
+	       "                        potentially expensive operation because it checks for\n"
+	       "                        every single process page if it contains only zeros but\n"
+               "                        it can significantly decrease the image size if many\n"
+               "                        such pages exist. It effectively replaces such pages\n"
+	       "                        which the kernel's zero-page on restore.\n"
```

Comment:
Would it be possible to move most of the details in the man page?
https://github.com/checkpoint-restore/criu/blob/criu-dev/Documentation/criu.txt

Reply:
Done